backends/tc/tc.def

```diff
@@ -446,7 +448,7 @@ class TCKey {
         emitValue = true;
     }
     toString {
-        std::string tckeyInstance = " " + keyAttribute + " " + keyName + " ptype bit" + Util::toString(bitwidth);
+        std::string tckeyInstance = " " + keyAttribute + " " + keyName + " ptype " + type;
```

Comment:
while you are here, I would suggest you change to `StrCat` here and to `StrAppend` two instances below. Not a big deal, but it is good to make cleanup in parallel :)

Reply:
addressed